security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisParameters.java

```diff
@@ -30,16 +30,103 @@
      * A configuration loader interface for the SecurityAnalysisParameters extensions loaded from the platform configuration
      * @param <E> The extension class
      */
-    public static interface ConfigLoader<E extends Extension<SecurityAnalysisParameters>> extends ExtensionConfigLoader<SecurityAnalysisParameters, E> {
+    public interface ConfigLoader<E extends Extension<SecurityAnalysisParameters>> extends ExtensionConfigLoader<SecurityAnalysisParameters, E> {
     }
 
-    public static final String VERSION = "1.0";
+    // VERSION = 1.0
+    // VERSION = 1.1 IncreasedViolationsParameters adding.
+    public static final String VERSION = "1.1";
 
     private static final Supplier<ExtensionProviders<ConfigLoader>> SUPPLIER =
-        Suppliers.memoize(() -> ExtensionProviders.createProvider(ConfigLoader.class, "security-analysis-parameters"));
+            Suppliers.memoize(() -> ExtensionProviders.createProvider(ConfigLoader.class, "security-analysis-parameters"));
 
     private LoadFlowParameters loadFlowParameters = new LoadFlowParameters();
 
+    public static class IncreasedViolationsParameters {
```

Comment:
I am not sure it is a good idea to add this object as it forbid you to set parameters using method chaining which I personally use very often when setting calculation parameters
```java
var params = new SecurityAnalysisParameters()
  .setIncreaseViolationHighVoltageProportionalThreshold(10)
  .setIncreaseViolationLowVoltageProportionalThreshold(13);
```

Or maybe:
```java
var params = new SecurityAnalysisParameters()
  .getIncreaseViolationParameters().setHighVoltageProportionalThreshold(10)
  .getIncreaseViolationParameters().setLowVoltageProportionalThreshold(13);
```

Not sure it is better.

Reply:
It adds another instruction but maybe you still can write it this way?
```java
var params = new SecurityAnalysisParameters()
    ....
    .set...();
params.getIncreaseViolationParameters()
    .setHighVoltageProportionalThreshold(10)
    .setLowVoltageProportionalThreshold(13);
```

Reply:
Ok why not. But in this case we have to create a sub object in the json too